ci/travis/install-dependencies.sh

```diff
@@ -202,6 +202,8 @@ install_toolchains() {
 install_dependencies() {
 
   install_bazel
+  "${WORKSPACE_DIR}"/ci/suppress_output bazel fetch "//:*" &  # Begin fetching packages in the background (network-bound)
```

Comment:
can we do multithreaded fetch? --jobs=20 or something

Reply:
Looking into this actually made me realize something.

To answer your question: it seems like Bazel actually can do this (`--loading_phase_threads`), though by default it seems to already parallelize to match the number of host CPU cores.

However, there's a more interesting fact that I see locally: `bazel fetch` seems to fetch entirely unnecessary packages for some bizarre reason. For example, when I run `bazel build -k "//:*"` locally, it begins compiling (everything is already fetched). But when I run `bazel fetch "//:*"`, it begins downloading darwin and linux Java tools on Windows.

This means this option might actually be causing more downloads to occur than would be normally necessary.

Let me take it out. Thanks for making me look into this! And for future reference, there's a chance that doing `bazel build --check_up_to_date "//:*"` will work instead, but we can look into that later.